sdk/textanalytics/azure-ai-textanalytics/samples/async_samples/sample_analyze_actions_async.py

```diff
@@ -12,8 +12,8 @@
 DESCRIPTION:
     This sample demonstrates how to submit a collection of text documents for analysis, which consists of a variety
     of text analysis actions, such as Entity Recognition, PII Entity Recognition, Linked Entity Recognition,
-    Sentiment Analysis, Key Phrase Extraction, or Extractive Text Summarization.  The response will contain results
-    from each of the individual actions specified in the request.
+    Sentiment Analysis, Key Phrase Extraction, or Extractive Text Summarization (not shown - see sample sample_extract_summary_async.py).
```

Comment:
nit: is providing a re-direct link would be better?

Reply:
we can't really do clickable links in our docstrings :(